splunkforwarder/elx/map.jinja

```diff
@@ -3,6 +3,7 @@
 
 package: splunkforwarder
 service: splunk
+service_opts: '--accept-license'
```

Comment:
let's remove the single quotes, they shouldn't be necessary

```suggestion
service_opts: --accept-license
```

Reply:
Removed